### PR TITLE
Jakob/newprizes

### LINKED ISSUE #103: Update prizes page with actual prizes
Once the prizes have been decided (also waiting on #76), update the prizes page to reflect that.

### PR DESCRIPTION
**Pull request by jakobreinwald**
Fixes #103 

- Added images to all the prizes and changed from last year's to this year's

### CODE REVIEW

File: src/components/PrizePage/PrizeHeader.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default function PrizeHeader() {
 				Prize Announcements Coming Soon!
```
Comment (TimothyGu) [inline src/components/PrizePage/PrizeHeader.js:30]:
```suggestion
				Prize Tracks
```

File: src/components/PrizePage/PrizeHeader.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default function PrizeHeader() {
...
 			</Typography>
 			<Typography variant='h3' component='h2' className={classes.lastYear}>
-				Last Year’s Prizes:
+				This Year’s Prizes:
 			</Typography>
```
Comment (TimothyGu) [inline src/components/PrizePage/PrizeHeader.js:34]: We should now delete this and the `lastYear` class.

File: src/components/PrizePage/Prizes.js [hunk 3]
```diff
@@ -157,7 +157,7 @@ export default function Prizes() {
 											{prize.prize}
 										</Typography>
 									</Box>
-									{/* <Img fixed={prizeImageMap.get(prize.img)} style={{ height: '128px' }} /> */}
+									{<Img fixed={prizeImageMap.get(prize.img)} style={{ height: '128px' }} /> }
```
Comment (TimothyGu) [inline src/components/PrizePage/Prizes.js:160]:
```suggestion
									<Img fixed={prizeImageMap.get(prize.img)} style={{ height: '128px' }} />
```

**Review by TimothyGu — CHANGES_REQUESTED** [review]

**Review by jamieliu386 — DISMISSED** [review]
In addition to what @TimothyGu already mentioned:

At a certain screen width, I get the below view. To fix it, we should have a transition between the three cards side by side to the single card view where there's two cards per row.

image

Also, this card looks a bit off because the text is too wide, so we should add some side padding to the text to make it wrap.

image

**Comment by TimothyGu** [comment]
I also see you created a `package-lock.json` file. That means you used npm to develop this repository, which is not what we use (we use Yarn instead).

**Review by TimothyGu — APPROVED** [review]

**Pull request merged**